src/main/java/lotto/model/constants/LottoWinningRank.java

```diff
@@ -0,0 +1,27 @@
+package lotto.model.constants;
+
+public enum LottoWinningRank {
+    FIRST(6, 2000000000),
```

Comment:
이렇게 숫자의 자리수가 많아지는 경우, `_`로 자리수를 구분시켜 줄 수 있습니다!
가독성이 훨씬 좋아져요
```suggestion
    FIRST(6, 2_000_000_000),
```

Reply:
좋은 방법입니다! 이렇게 해도 integer에 저장이 되는군요.. 하나 더 배웠습니다